src/index.js

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
 import App from './components/App';
 import ErrorBoundary from './components/common/ErrorBoundary';
+import reducers from './reducers';
+import { createStore, applyMiddleware, compose } from 'redux';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const store = createStore(reducers, composeEnhancers(
+    applyMiddleware(thunk)
+));
```

Comment:
it's a common practice to init store in separate file and import it here.

One more thing is that `window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__` is going to be a part of `prod` bundle, but do we need it there/. You can try to make use of NODE_ENV process env variable from the first task to exclude it for `prod` build